test/profiler/test_profiler.py

```diff
@@ -2081,6 +2084,84 @@ def validate_forked_json(profiler):
         else:
             os.waitpid(pid, 0)
 
+    @unittest.skipIf(IS_WINDOWS, "torch.compile does not support WINDOWS")
```

Comment:
maybe put this in test/inductor/test_profiler.py?

Reply:
ohhh i didn't know about this file, let me move it there!